gooddata-dbt/gooddata_dbt/dbt/cloud.py

```diff
@@ -17,6 +20,52 @@
 class DbtCloudBase:
     base_v2: str = attrs.field(init=False, default="https://cloud.getdbt.com/api/v2")
     base_v3: str = attrs.field(init=False, default="https://cloud.getdbt.com/api/v3")
+    base_graphql: str = attrs.field(init=False, default="https://metadata.cloud.getdbt.com/graphql")
+
+    graphql_applied_models = """
+        query AppliedModels($environmentId: BigInt!, $first: Int!) {
+            environment(id: $environmentId) {
+              applied {
+                models(first: $first) {
+                  edges {
+                    node {
+                      name
+                      uniqueId
+                      materializedType
+                      executionInfo {
+                        lastSuccessRunId
+                        executionTime
+                        executeStartedAt
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        """
+    graphql_model_historical_runs = """
+        query modelHistoricalRuns($environmentId: BigInt!, $modelId: String!, $first: Int!) {
+          environment(id: $environmentId) {
+            applied {
+              modelHistoricalRuns(
+                uniqueId: $modelId
+                lastRunCount: $first
+              ) {
+                runId # Get historical results for a particular model
+                runGeneratedAt
+                executionTime # View build time across runs
+                status
+                #tests {
+                #  name
+                #  status
+                #  executeCompletedAt
+                #} # View test results across runs
```

Comment:
Do we want to include these comment attributes?

Reply:
Removed. Can be added any time later.